gr-blocks/python/blocks/qa_msg_pair_to_var.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# Copyright 2023 Marcus Müller
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+
+
+from gnuradio import blocks
+from gnuradio import gr_unittest
+from gnuradio import gr
+import pmt
+import sys
+import time
+
+
+class test_msg_pair_to_var(gr_unittest.TestCase):
+
+    def setUp(self):
+        self.backend = gr.dictionary_logger_backend()
+        gr.logging().add_default_sink(self.backend)
+        self.tb = gr.top_block(catch_exceptions=False)
+
+    def tearDown(self):
+        self.tb = None
+
+    def test_calling(self):
+        canary = 0xDEAFBEEF
+        self.test_val = 0
+
+        def test_f(value: int = 0):
+            self.test_val = value
+
+        dut = blocks.msg_pair_to_var(test_f)
+        test_msg = pmt.cons(pmt.intern("foo"), pmt.from_uint64(canary))
+        src = blocks.message_strobe(test_msg, 10)
+        self.tb.msg_connect(*(src, pmt.intern("strobe")),
+                            *(dut, blocks.msg_pair_to_var.IN_PORT))
+        self.tb.start()
+        time.sleep(15e-3)
```

Comment:
The short wait time here readily causes test failures on my system if I add some CPU load with `stress -c 8`.